types/enginko/MCF-LW12MET/io.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$id": "https://akenza.io/enginko/EGK-LW12MET/IO.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "IO",
```

Comment:
possibly find a more speaking name

Reply:
I could do Input/Output
In other devices i also should change this then